src/app/components/App/AppContainer.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import * as firebase from "firebase";
+import { connect } from 'react-redux';
+import { setAuthState } from '../../redux/actions.js';
+import App from './App.js';
+
+var config = {
```

Comment:
Technically it will be okay once auth gets setup. If we want to keep it in fine. But we will have to make sure all db authentication is setup.